app/controllers/telegram_division_controller.rb

```diff
@@ -24,6 +24,7 @@ def summary
   private
 
   def summary_report
+    a = "dasa"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.